Sofa/framework/Type/src/sofa/type/config.h.in

```diff
@@ -36,7 +36,7 @@
 #define SOFA_ATTRIBUTE_DEPRECATED__REBIND()
 #else
 #define SOFA_ATTRIBUTE_DEPRECATED__REBIND() \
-    SOFA_ATTRIBUTE_DEPRECATED( \
+    SOFA_ATTRIBUTE_DISABLED( \
```

Comment:
SOFA_ATTRIBUTE_DEPRECATED__REBIND should be renamed to
SOFA_ATTRIBUTE_DISABLED__REBIND and the code base updated